tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/NewApprenticeshipProvider/HidePassedNotificationTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿using Dfc.CourseDirectory.WebV2.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Dfc.CourseDirectory.WebV2.Tests.FeatureTests.NewApprenticeshipProvider
+{
+    public class HidePassedNotificationTests : TestBase
+    {
+        public HidePassedNotificationTests(CourseDirectoryApplicationFactory factory)
+            : base(factory)
+        {
+        }
+
```

Comment:
Need a test to check Helpdesk users get a Forbidden response

Reply:
done.

Reply:
done